python/MooseDocs/extensions/gitutils.py

```diff
@@ -68,8 +69,11 @@ def createToken(self, parent, info, page):
         if not inline:
             raise exceptions.MooseDocsException("The '!git submodule-hash' command is an inline level command, use '[!git!submodule-hash](name)' instead.")
 
-        name =  info['inline']
+        name = info['inline']
         status = mooseutils.git_submodule_info(MooseDocs.ROOT_DIR, '--recursive')
+        # if (name == 'moose') and any([repo.endswith('moose') for repo in status.keys()]) and (status['moose'][0] == '-'):
+        #     logging.getLogger(__name__).warning("The 'moose' submodule exists but has not been initialized.")
+
```

Comment:
Can you explain why you are removing this? Also, if it is not needed then just delete it.

Reply:
@aeslaughter I'm actually not removing anything here. I added it, but then commented it out. I'll just delete it.